<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pacient_Data_in_Menu</class>
 <widget class="QWidget" name="Pacient_Data_in_Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyLabelAnimated" name="pb_aceptar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>287</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border: 2px solid #E3E3E3;
     border-radius: 5px;
     color: #E3E3E3
 }</string>
   </property>
   <property name="text">
    <string>ACEPTAR</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>84</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	 border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_weight">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>84</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Nombre: |</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>84</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro Cond Bold</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	 border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_CI_HC">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>47</y>
     <width>245</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>84</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Edad: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>47</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Sexo: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>84</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Talla: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>47</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>CI:/HC: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_sexo">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>47</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_edad">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>84</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_height">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>84</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>84</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #24292E
 }</string>
   </property>
   <property name="text">
    <string>Peso: |</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="le_nombre">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>10</y>
     <width>434</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyPlainTextEdit" name="pt_comentarios">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>521</width>
     <height>120</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Pro</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <zorder>pb_aceptar</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_11</zorder>
  <zorder>label_8</zorder>
  <zorder>label_4</zorder>
  <zorder>label_10</zorder>
  <zorder>le_weight</zorder>
  <zorder>label_14</zorder>
  <zorder>le_height</zorder>
  <zorder>label_9</zorder>
  <zorder>le_edad</zorder>
  <zorder>le_CI_HC</zorder>
  <zorder>le_nombre</zorder>
  <zorder>le_sexo</zorder>
  <zorder>pt_comentarios</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">myplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
